client/src/components/OrderListEntity.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import style from '../style/style.css.jsx';
+
+class OrderListEntity extends React.Component {
+    constructor(props) {
+      super(props);
+  
+      this.state = {
+        visiable: false,
+      };
+    }
+
+    handleAddToCart(e) {
+        e.preventDefault();
+        alert('will be added to bag');
+    }
+
+    onMouseEnter() {
+        this.setState({
+            visiable: true
+        });
+    }
+
+    onMouseLeave() {
+        this.setState({
+            visiable: false
+        });
+    }
+    
+    render() {
+        return (
+        <div className="order-list-entity" style={style.orderListEntity} onMouseEnter={this.onMouseEnter.bind(this)} onMouseLeave={this.onMouseLeave.bind(this)}>
+            {this.props.item['name'] + ' + '}
+            {
+                this.state.visiable ?
+                    <form className="tooltiptext" style={style.tooltiptext}>
+                        <div style={style.tooltiptextHeader}><span style={style.userName}>{this.props.item['name']}</span><span>${this.props.item['price']}</span></div>
+                        {
+                            this.props.item['content'].length === 0?
```

Comment:
The question mark should be at the start of the second line below, and the div should be after the colon. See https://github.com/airbnb/javascript#comparison--nested-ternaries

Reply:
i will fix that